src/app/components/applications/SSV/MyAccount/components/EditOperatorDetails/EditOperatorDetails.tsx

```diff
@@ -47,10 +48,14 @@ const EditOperatorDetails = () => {
         if (!isNotValidity) {
             let payload = metadataStore.createMetadataPayload();
             let rawDataToValidate: any = [];
-            Object.values(payload).map(value =>
-                rawDataToValidate.push(value),
-            );
-            rawDataToValidate = rawDataToValidate.join('|');
+            Object.keys(payload).forEach(field => {
+                if (payload[field]) {
+                    const newItem =
+                        field === 'logo' ? sha256(payload[field]) : payload[field];
+                    rawDataToValidate.push(newItem);
+                }
+            });
+            rawDataToValidate = rawDataToValidate.join(',');
```

Comment:
This is not correct implementation.
In requirements was not anything about removing sequential compilation of the data with filed name array.

Reply:
done

Reply:
done